prisma/schema.prisma

```diff
@@ -222,3 +222,11 @@ model Deposit {
   @@map("deposits")
   @@unique([transaction_hash, graffiti], name: "uq_deposits_on_transaction_hash_and_graffiti", map: "uq_deposits_on_transaction_hash_and_graffiti")
 }
+
+model DepositHead {
+    id                Int                @id
```

Comment:
do we need this?

Reply:
yes so we know to update only the single head object with a constant id (1)